app/scripts/directives/mobileServiceClientAssociations.js

```diff
@@ -27,10 +27,12 @@
     ctrl.$onInit = function() {
       ctrl.heading = 'Mobile Clients';
       ctrl.type = 'client';
+      ctrl.hasResources = !_.isEmpty(ctrl.mobileClients);
```

Comment:
Can this be removed with this? https://github.com/openshift/origin-web-console/pull/2903/files#diff-6ea6053de312eed10ac45db60433ffc2R165 the `row.hasMobileClients` bit?

Reply:
If you mean remove it completely I don't think that would work because `ng-if="$ctrl.hasResources"`  would become false